src/main/kotlin/io/github/orangain/prettyjsonlog/logentry/Timestamp.kt

```diff
@@ -49,7 +49,7 @@ sealed interface Timestamp {
     }
 }
 
-private val timestampKeys = listOf("timestamp", "time", "@timestamp", "ts", "@t", "Timestamp")
```

Comment:
@LeeGilbert Thank you for reporting the issue and sending the pull request!

According to [the document of Cloud Logging](https://cloud.google.com/logging/docs/agent/logging/configuration?hl=en#timestamp-processing), the `timestampNanos` field will be used together with the `timestampSeconds` field, and the `timestampNanos` represents only fractional seconds of the timestamp.

Therefore, to parse these fields correctly, we need to extend the `extractTimestamp` function and the `timestampKeys` variable to handle multiple fields at once.

In addition, adding a test case would be appreciated.

-----
If you find this a little troublesome, I can implement it as the owner of the repository. Please let me know if you would like me to do this.

Reply:
My bad, I've updated the push with the change to parse both timestampSeconds, and the timestampNanos plus a test. I'm not setup to compile locally, hopefully its ok.